src/include/function/aggregate_function.h

```diff
@@ -123,12 +123,12 @@ class AggregateFunctionUtil {
     static std::unique_ptr<AggregateFunction> getAvgFunc(const std::string name,
         common::LogicalTypeID inputType, common::LogicalTypeID resultType, bool isDistinct);
     static std::unique_ptr<AggregateFunction> getMinFunc(
-        const common::LogicalType& inputType, bool isDistinct);
+        common::LogicalTypeID inputType, bool isDistinct);
```

Comment:
I think this is fine in this PR. But in the long run, all functions (scalar, aggregation, table, ...) should have the same base class `Function` whose parameters are `LogicalType` rather than `LogicalTypeID` which means we will be using `LogicalType` almost everywhere for function related code path.

Reply:
Mostly it seems weird to me to pass around versions of `LogicalType` which have no `extraTypeInfo`, even for complex types, particularly when `LogicalTypeID` is functionally identical.